deploy/ansible/roles/iroha-k8s/files/conf/config.sample

```diff
@@ -1,10 +1,11 @@
 {
-  "block_store_path" : "/tmp/block_store/",
+  "block_store_path" : "/opt/block_store/",
```

Comment:
grep said to me that there's 6 occurrences of `"/tmp/block_store/"` in deploy folder.
@bakhtin may you confirm, that all required ones are fixed?

Reply:
Could you please leave it in /tmp for now? We want to keep block store location uniform across the repo. But I agree that /tmp is not the best location for any data that should be persisted.

Reply:
No problem, I've reverted my change